sonarts-core/tests/rules/argumentsOrderRule/argumentsOrderRule.lint.ts

```diff
@@ -0,0 +1,52 @@
+export function sameType(a: number, b: number, c = 3) {}
+
+const a = 1, b = 2, c = 3, d = 4;
+
+sameType(a, b, c);
+sameType(a, b, d);
+sameType(d, d, d);
+sameType(a, a, a);
+sameType(42, a, c);
+sameType(a, d, b);
+
+sameType(b, a, d);
+//       ^^^^^^^ {{Parameters have the same names but not the same order as the arguments.}}
```

Comment:
I think it would be nice to 
- either mention in the message what parameters are swapped 
- or to highlight the function name, having secondary locations for the swapped parameters

what do you think?